assets/component-search.css

```
@@ -1,32 +1,71 @@
 .search__input.field__input {
-  padding-right: 5rem;
+  padding-right: 9.2rem;
 }
 
 .search__button {
   right: var(--inputs-border-width);
   top: var(--inputs-border-width);
 }
 
-.search__button:focus-visible {
-  background-color: rgb(var(--color-background));
-  z-index: 2;
+.reset__button {
+  right: calc(var(--inputs-border-width) + 4.4rem);
+  top: var(--inputs-border-width);
+}
+
+/* Show a divider between reset and submit buttons */
+.reset__button:not(:focus-visible)::after {
+  border-right: 0.1rem solid rgba(var(--color-foreground), 0.2);
+  display: block;
+  height: calc(100% - 1.6rem);
+  content: "";
+  position: absolute;
+  right: 0;
 }
 
-.search__button:focus {
+.search__button:focus-visible,
+.reset__button:focus-visible,
+.search__button:focus,
+.reset__button:focus {
```

One thing that comes to mind at quick glance is that we keep those separated `:focus-visible` declarations and `:focus` ones as a browser that doesn't support `focus-visible` like older versions of Safari won't read the whole thing if it doesn't support one selector. 
```suggestion
.search__button:focus-visible,
.reset__button:focus-visible {
  ...
}

.search__button:focus,
.reset__button:focus {
 ...
 }
```

---

TIL